### PR TITLE
WIP: Notched boxplot fix

### PR DESCRIPTION
**Pull request by grero**
This is my initial attempt at addressing #59. Any comments and corrections more than welcome : )

### CODE REVIEW

**Comment by grero** [comment]
This currently only works for a single plot. The way I constructed the notched boxes doesn't play nice when plotting multiple distributions.

**Comment by piever** [comment]
Thanks! I would actually prefer to not have a committed Manifest.toml as part of the package. Could you please remove it and add it to .gitignore?

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+FreqTables = "da1fdf0e-e0ff-5433-a45f-9bb5ff651cb1"
+IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
+KernelDensity = "5ab0869b-81aa-558d-bb23-cbf5423bbe9b"
+Loess = "4345ca2d-374a-55d4-8d30-97f9976e7612"
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
+NamedArrays = "86f7a689-2022-50b4-a561-43c23ac3c673"
+Observables = "510215fc-4207-5dde-b226-833fc4488ee2"
+OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```
Comment (piever) [inline Project.toml]: Not a dependency

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+[deps]
+AbstractPlotting = "537997a7-5e4e-5d89-9595-2241ea00577e"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+FreqTables = "da1fdf0e-e0ff-5433-a45f-9bb5ff651cb1"
+IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
+KernelDensity = "5ab0869b-81aa-558d-bb23-cbf5423bbe9b"
+Loess = "4345ca2d-374a-55d4-8d30-97f9976e7612"
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
```
Comment (piever) [inline Project.toml]: Makie is not a StatsMakie dependency

File: Project.toml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+NamedArrays = "86f7a689-2022-50b4-a561-43c23ac3c673"
+Observables = "510215fc-4207-5dde-b226-833fc4488ee2"
+OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```
Comment (piever) [inline Project.toml]: Should only be a "extras", see for example [here](https://github.com/JuliaDocs/Documenter.jl/blob/master/Project.toml#L16)

**Comment by piever** [comment]
> This currently only works for a single plot. The way I constructed the notched boxes doesn't play nice when plotting multiple distributions.

I don't really understand what you mean. Could you give examples (possibly with screenshots) of plots that work and plots that don't work?

File: Project.toml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+name = "StatsMakie"
+uuid = "65254759-4cff-5aa5-8326-61ce017a8c70"
+version = "0.0.1"
```
Comment (piever) [inline Project.toml]: current version is 0.0.3

File: Project.toml (hunk 3 above)
Comment (grero) [reply Project.toml]: Fixed in c6325d407354feb3c18f1097444bbdd4b64fe11a

File: Project.toml (hunk 4 above)
Comment (grero) [reply Project.toml]: Fixed in 9535a472be34677a683d4bc09de698a314083716

File: Project.toml (hunk 2 above)
Comment (grero) [reply Project.toml]: Fixed in bdf4b87e41ea5bd1c6412cf37d817c61f56b3b65

File: Project.toml (hunk 1 above)
Comment (grero) [reply Project.toml]: This snuck in as I was working on this. Fixed  in 301959131ffb4a01922d7e87d96f6ba0a277e1cf (where I accidentally referred to Revise as Refactor for some reason)

**Comment by grero** [comment]
This now works for multiple distributions as well. This is what it looks likes using the synthetic data in the test suite:

test_img

**Comment by piever** [comment]
Looks good, thanks a lot!

**Pull request merged**